includes/class-llms-block-library.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * LLMS_Block_Library class file.
+ *
+ * @package LifterLMS/Classes
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Load the LifterLMS block library.
+ *
+ * @sinc [version]
+ */
+class LLMS_Block_Library {
+
+	/**
+	 * Constructor.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function __construct() {
+
+		add_action( 'init', array( $this, 'register' ) );
+
+	}
+
+	/**
+	 * Retrieves a list of blocks to register.
+	 *
+	 * @since [version]
+	 *
+	 * @return string[] A list of directory paths that can individually be passed to `register_block_type()`.
+	 */
+	private function get_blocks() {
+
+		$blocks = array(
```

Comment:
@eri-trabiccolo I originally had a filter on the return but then I thought better of it because, simply, it's basically list of things that can be easily passed into `register_block_type()`. I suppose that there might be some quality of life for us in our add-ons to save a few lines of code though and maybe some developer might want to use it...But... also... who cares? I'll add it back in.

Reply:
No... I'm not filtering it. This registers our core blocks, not external blocks. External blocks can be registered on their own and if someone wants to remove a core LLMS block they can do so by deregistering the block using the WP_Block_Registry class.

My reasoning here is that we'd need to create an exception handler for users to define a different namespace for the dupcheck in the `register()` method.

All in all if someone wants to register their own block... the API already exists in WP and they don't need to use our API to do so.

Reply:
Mmm I did a mess I think. I meant to add that comment to the list of fonts...

Reply:
Here: https://github.com/gocodebox/lifterlms/pull/1899/files#diff-572a859272cff978c2699df82291b4a0605f707085cb2d3bca0a058f30589ccaR162

?

Reply:
Yeah 😂